cmd/lk/sip.go

```diff
@@ -271,14 +311,16 @@ func listSipInboundTrunk(ctx context.Context, cmd *cli.Command) error {
 		"SipTrunkID", "Name", "Numbers",
 		"AllowedAddresses", "AllowedNumbers",
 		"Authentication",
+		"Encryption",
 		"Headers",
 		"Metadata",
 	}, func(item *livekit.SIPInboundTrunkInfo) []string {
 		return []string{
 			item.SipTrunkId, item.Name, strings.Join(item.Numbers, ","),
 			strings.Join(item.AllowedAddresses, ","), strings.Join(item.AllowedNumbers, ","),
 			userPass(item.AuthUsername, item.AuthPassword != ""),
-			fmt.Sprintf("%v, %v", item.Headers, item.HeadersToAttributes),
+			strings.TrimPrefix(item.MediaEncryption.String(), "SIP_MEDIA_ENCRYPT_"),
+			printHeaderMaps(item.Headers, item.HeadersToAttributes),
```

Comment:
IDK how big this data is but is it readable in this format inside a table cell?

Reply:
Changed the format for these maps. They will now print each `k=v` on a separate line.